opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { FunctionComponent } from 'react';
+import { NarrativeLine_node$data } from '@components/techniques/narratives/__generated__/NarrativeLine_node.graphql';
+import ToolBar from '@components/data/ToolBar';
+import ExportContextProvider from '../../../utils/ExportContextProvider';
+import { NarrativeLineDummy } from './narratives/NarrativeLine';
+import NarrativesLines, { narrativesLinesQuery } from './narratives/NarrativesLines';
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+import ListLines from '../../../components/list_lines/ListLines';
+import Security from '../../../utils/Security';
+import { KNOWLEDGE_KNPARTICIPATE, KNOWLEDGE_KNUPDATE } from '../../../utils/hooks/useGranted';
+import NarrativeCreation from './narratives/NarrativeCreation';
+import { emptyFilterGroup, useBuildEntityTypeBasedFilterContext } from '../../../utils/filters/filtersUtils';
+import { useFormatter } from '../../../components/i18n';
+import Breadcrumbs from '../../../components/Breadcrumbs';
+import { NarrativesLinesPaginationQuery, NarrativesLinesPaginationQuery$variables } from './narratives/__generated__/NarrativesLinesPaginationQuery.graphql';
+import useEntityToggle from '../../../utils/hooks/useEntityToggle';
+import useHelper from '../../../utils/hooks/useHelper';
+
+const LOCAL_STORAGE_KEY = 'narratives';
+
+const Narratives: FunctionComponent = () => {
+  const { t_i18n } = useFormatter();
+  const { isFeatureEnable } = useHelper();
+  const FAB_REPLACED = isFeatureEnable('FAB_REPLACEMENT');
+  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage < NarrativesLinesPaginationQuery$variables >(
```

Comment:
```suggestion
  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<NarrativesLinesPaginationQuery$variables>(
```
we usually don't put spaces their